Jenkinsfile

```diff
@@ -5,6 +5,18 @@ import com.tenable.jenkins.common.*
 import com.tenable.jenkins.Constants
 import com.tenable.jenkins.builds.snyk.*
 import com.tenable.jenkins.builds.nexusiq.*
+    
+
+
+def addParameters(final script, def theParams) {
+    
```

Comment:
This parameter should only be added on master.

i.e.

if (script.env.BRANCH_NAME == 'master') {
....